apps/web-mzima-client/src/app/settings/surveys/survey-task/survey-task.component.ts

```diff
@@ -356,4 +365,35 @@ export class SurveyTaskComponent implements OnInit, OnChanges {
   public changeLabel(value: string) {
     this.errorFieldChange.emit(value.trim().length === 0);
   }
+
+  public trackHoverEvent(event: Event, field: any) {
+    /* --------------------------------------------
+      Show an info button on survey fields on hover
+    ---------------------------------------------*/
+    this.fieldHover = event.type === 'mouseenter';
+    this.fieldId = field.id;
+  }
```

Comment:
This works, with the mouse. Is there a way to make this available also for keyboard-users? 😬 I think perhaps with focus and blur-event + aria-labels? 🤔